app/views/scheduler/index.html.haml

```diff
@@ -9,8 +9,11 @@
           = text_field_tag :course_title, nil, placeholder: 'Add course instance', class: 'form-control'
 
         = form_for :semester, url: scheduler_path, method: :get, remote: true, html: {id: :semester_form} do |f|
-          = f.select :semester, %w(Fall Spring Summer).map { |str| [str, str.downcase] }
-          = f.select :year, options_for_select(2008..DateTime.now.year, DateTime.now.year)
+          = f.select :semester, options_for_select( |
+            %w(Fall Spring Summer).map {|str| [str, str.downcase]}, |
```

Comment:
The pipe on the second line is required - it's a quirk of the way the pipes work, but since I'm breaking the line from line 12 to 13, both 12 and 13 need pipes. Because there is a comma at the end of 13, that's why 14 doesn't need a pipe. See: http://stackoverflow.com/a/26253127

And you're correct about the fact that the text shows up downcased, my bad, that didn't stick out to me. But yeah of course I've been verifying locally, I wrestled with the interpreter for about an hour before I finally figured out that pipes are required for line overflows in haml. I just wish I had a way to get Hound feedback locally.